src/ucp/rndv/rndv.c

```diff
@@ -1355,11 +1400,12 @@ UCS_PROFILE_FUNC_VOID(ucp_rndv_receive, (worker, rreq, rndv_rts_hdr, rkey_buf),
         goto err;
```

Comment:
btw, this is incorrect `goto`, we have to do UNBLOCK as well

Reply:
it goes to `err` and then to `out` from `err`:
https://github.com/openucx/ucx/blob/b7f8cc2dce80fd654356fbec527d957f0fedd520/src/ucp/rndv/rndv.c#L1449